tests/core/utilities/test_valid_transaction_params.py

```diff
@@ -38,20 +38,49 @@ def test_assert_valid_transaction_params_invalid_param():
 
 
 def test_extract_valid_transaction_params():
+    ''' baseline test for clean param dict '''
     input = {
         'from': '0x0',
         'to': '0x0',
         'gas': 21000,
         'gasPrice': 5000000,
-        'value': 1,
         'data': '0x0',
+        'value': 1,
```

Comment:
Moving `value` here wasn't meaningful, right? If it affects the test somehow, we've got a bigger problem.